phpBB/includes/compatibility_globals.php

```diff
@@ -29,7 +29,7 @@ function register_compatibility_globals()
 {
 	global $phpbb_container;
 
-	global $cache, $phpbb_dispatcher, $request, $user, $auth, $db, $config, $phpbb_log;
+	global $cache, $phpbb_dispatcher, $request, $user, $auth, $db, $config, $language, $phpbb_log;
```

Comment:
Yeah nothing relevant as I said, then for the same reason `$language` should go after `$user` in this line. ;)
Again, it's not important nor relevant. Thanks.